backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/ComputingMethodType.java

```diff
@@ -74,15 +54,22 @@ public enum ComputingMethodType implements ReferenceListAwareEnum
 	AverageAddedValueOnShippedQuantity(X_ModCntr_Type.MODULARCONTRACTHANDLERTYPE_AverageAddedValueOnShippedQuantity),
 	AddValueOnInterim(X_ModCntr_Type.MODULARCONTRACTHANDLERTYPE_AddValueOnInterim),
 	SubtractValueOnInterim(X_ModCntr_Type.MODULARCONTRACTHANDLERTYPE_SubtractValueOnInterim),
+	SalesStorageCost(X_ModCntr_Type.MODULARCONTRACTHANDLERTYPE_SalesStorageCost),
+	Sales(X_ModCntr_Type.MODULARCONTRACTHANDLERTYPE_Sales),
+	SalesAV(X_ModCntr_Type.MODULARCONTRACTHANDLERTYPE_SalesAV),
```

Comment:
Note: we sould also rename the AddedValueOnRaw and AddedValueOnProcessed intro PurchaseAddedValueOnRaw  and PurchaseAddedValueOnProcessed  but it's up to your aesthetics taste, since we know that on sales there's no distinction between raw and processed.